java/feature-layer-definition-expression/README.md

```diff
@@ -1,18 +1,34 @@
-# Feature Layer Definition Expression
-Limit which features to display on the map.
+# Feature layer definition expression
 
-![Feature Layer Definition Expression App](feature-layer-definition-expression.png)
+Limit the features displayed on a map with a definition expression.
+
+![Image of feature layer definition expression](feature-layer-definition-expression.png)
+
+## Use case
+
+Set a definition expression to filter out the features to be displayed. You might filter a dataset of tree quality selecting for only those trees which require maintenance or are damaged.
 
 ## How to use the sample
-Use the buttons in the bottom toolbar to apply or reset definition expression.
+
+Press the 'Apply Expression' button to limit the features requested from the feature layer to those specified by the SQL query definition expression. Click the 'Reset' button to remove the definition expression on the feature layer, which returns all the records.
 
 ## How it works
 
-Set the definition expression property on a feature layer using an SQL where clause to limit which features are displayed on the map.
+1. Create a service feature table from a URL.
+2. Create a feature layer from the service feature table.
+3. Set the limit of the features on your feature layer using the `DefinitionExpression`.
 
 ## Relevant API
+
+* DefinitionExpression
```

Comment:
there is still a reference to DefinitionExpression here

Reply:
thanks!